common/lib/dependabot/shared_helpers.rb

```diff
@@ -122,7 +122,10 @@ def self.run_helper_subprocess(command:, function:, args:, env: nil,
     # rubocop:enable Metrics/MethodLength
 
     def self.excon_middleware
-      Excon.defaults[:middlewares] + [Excon::Middleware::RedirectFollower]
+      Excon.defaults[:middlewares] + [
+        Excon::Middleware::RedirectFollower,
```

Comment:
I might be wrong but I think we want the redirect to happen before decompressing, starting from the right. At least from the examples I've seen. Already fixed here just forgot to merge it: https://github.com/dependabot/dependabot-core/pull/1483

Reply:
My bad.  I was running into this problem while working on https://github.com/dependabot/feedback/issues/738, hence why I did the fix.  I just saw your comment and didn't realize you had a PR already.